Marlin/Configuration_adv.h

```diff
@@ -847,47 +847,59 @@
 
 /**
  * Advanced Pause
- * Experimental feature for filament change support and for parking the nozzle when paused.
- * Adds the GCode M600 for initiating filament change.
- * If PARK_HEAD_ON_PAUSE enabled, adds the GCode M125 to pause printing and park the nozzle.
+ *  M600/Runout script : Filament change feature.
+ *  M125 to pause printing.
+ *    Park the nozzle
+ *    Apply Fanspeed and dwell to cooled purged filament
+ *    Retract/Recover before/after travel
+ *    Requires : LCD display + FILAMENT_RUNOUT_SCRIPT .
+ * 
+ * Spool_Swap 
+ *  Tool change after filament runout.
+ *  M600: T[int]: 0=disable : 1/2/3/4/5 Enable the max number of swapping
+ *    Finish a spool and begin another by swapping to next extruder automaticly 
+ *    Copy Temp + Flow + Fwretract
+ *    Requires (2 extruders + 2 runout sensors ) at min
+ *    
+ *    Normal M600 run after max swap reached
  *
- * Requires an LCD display.
- * This feature is required for the default FILAMENT_RUNOUT_SCRIPT.
  */
-//#define ADVANCED_PAUSE_FEATURE
-#if ENABLED(ADVANCED_PAUSE_FEATURE)
```

Comment:
USE MY NAMES AND RENAME YOUR OWN MARLIN 2.0 
'My names are serious and can be merged ' but i'm sure you will change something lollllllllllllll